clang/lib/Sema/SemaHLSL.cpp

```diff
@@ -1698,18 +1698,27 @@ static bool CheckVectorElementCallArgs(Sema *S, CallExpr *TheCall) {
   return true;
 }
 
-static bool CheckArgsTypesAreCorrect(
+bool CheckArgTypeIsIncorrect(
```

Comment:
This switches the naming pattern from the existing convention.  Why?  It makes me think: "Make sure the arg type is incorrect".  I know that the other functions return true on failure and false on success, but that's just the standard convention for sema checking functions.